File: app/src/main/java/com/example/ljl/LogUtil.java
package com.example.ljl;

import android.util.Log;

public class LogUtil {
    private static boolean debuge_v = true;

    private static boolean debuge_d = true;

    private static boolean debuge_i = true;

    private static boolean debuge_w = true;

    private static boolean debuge_e = true;

    private static final String MAINTAG = "OTA:";

    /**
     * vlog.
     * @param tag tag
     * @param message mes
     */
    public static final void vlog(String tag, String message) {
        if (debuge_v) {
            Log.v(MAINTAG + tag, message);
        }
    }

    /**
     * dlog.
     * @param tag tag
     * @param message mes
     */
    public static final void dlog(String tag, String message) {
        if (debuge_d) {
            Log.d(MAINTAG + tag, message);
        }
    }

    /**
     * ilog.
     * @param tag tag
     * @param message mes
     */
    public static final void ilog(String tag, String message) {
        if (debuge_i) {
            Log.i(MAINTAG + tag, message);
        }
    }

    /**
     * wlog.
     * @param tag tag
     * @param message mes
     */
    public static final void wlog(String tag, String message) {
        if (debuge_w) {
            Log.w(MAINTAG + tag, message);
        }
    }

    /**
     * elog.
     * @param tag tag
     * @param message mes
     */
    public static final void elog(String tag, String message) {
        if (debuge_e) {
            Log.e(MAINTAG + tag, message);
        }
    }
}

File: tspManager/src/main/java/com/zk/tspmanager/utils/LogUtil.kt
package com.zk.tspmanager.utils

import android.util.Log

object LogUtil {
    private const val debuge_v = true
    private const val debuge_d = true
    private const val debuge_i = true
    private const val debuge_w = true
    private const val debuge_e = true
    private const val MAINTAG = "TspManager:"

    /**
     * vlog.
     * @param tag tag
     * @param message mes
     */
    fun vlog(tag: String, message: String?) {
        if (debuge_v) {
            Log.v(MAINTAG + tag, message!!)
        }
    }

    /**
     * dlog.
     * @param tag tag
     * @param message mes
     */
    fun dlog(tag: String, message: String?) {
        if (debuge_d) {
            Log.d(MAINTAG + tag, message!!)
        }
    }

    /**
     * ilog.
     * @param tag tag
     * @param message mes
     */
    fun ilog(tag: String, message: String?) {
        if (debuge_i) {
            Log.i(MAINTAG + tag, message!!)
        }
    }

    /**
     * wlog.
     * @param tag tag
     * @param message mes
     */
    fun wlog(tag: String, message: String?) {
        if (debuge_w) {
            Log.w(MAINTAG + tag, message!!)
        }
    }

    /**
     * elog.
     * @param tag tag
     * @param message mes
     */
    fun elog(tag: String, message: String?) {
        if (debuge_e) {
            Log.e(MAINTAG + tag, message!!)
        }
    }
}